graph/src/env/mod.rs

```diff
@@ -362,6 +356,18 @@ impl EnvVars {
             .filter(|x| !x.is_empty())
             .collect()
     }
+    pub fn reorg_threshold(&self) -> i32 {
+        // The default reorganization (reorg) threshold is set to 250.
+        // For testing purposes, we need to set this threshold to 0 because:
+        // 1. Many tests involve reverting blocks.
+        // 2. Blocks cannot be reverted below the reorg threshold.
+        // Therefore, during tests, we want to set the reorg threshold to 0.
+        if *TEST_WITH_NO_REORG.lock().unwrap() {
+            0
+        } else {
+            self.reorg_threshold
+        }
+    }
```

Comment:
This is pretty minor, but AFAICT, we only need the ability to override the reorg threshold when debug assertions is on. Could we therefore have a variant of this function for `#[cfg(not(debug_assertions))]` that doesn't touch the mutex?